storage/remote/codec_test.go

```diff
@@ -336,7 +336,7 @@ func TestMetricTypeToMetricTypeProto(t *testing.T) {
 }
 
 func TestDecodeWriteRequest(t *testing.T) {
```

Comment:
Can we test building request with proto TimeSeries AND populating samples (as it was) and marshal the same data using our marshalling and compare binary (I think we expect it to be 1:1?).